test/msg.jl

```diff
@@ -0,0 +1,21 @@
+if VERSION >= v"0.5-"
+    using Base.Test
+else
+    using BaseTestNext
+    const Test = BaseTestNext
+end
+
```

Comment:
These `using` statements don't need to be here since they are in `runtests.jl` ... I don't think we need to worry about the case where people want to runt the different tests individually.